master/internal/db/postgres_experiments.go

```diff
@@ -937,18 +937,20 @@ func ActiveLogPolicies(
 // ExperimentTotalStepTime returns the total elapsed time for all allocations of the experiment
 // with the given ID. Any step with a NULL end_time does not contribute. Elapsed time is
 // expressed as a floating point number of seconds.
-func (db *PgDB) ExperimentTotalStepTime(id int) (float64, error) {
+func (db *PgDB) ExperimentTotalStepTime(ctx context.Context, id int) (float64, error) {
```

Comment:
We have move towards using package level functions over using the db method.

```suggestion
func ExperimentTotalStepTime(ctx context.Context, id int) (float64, error) {
```

Reply:
I have incorporated using package level functions in current version.